test/test_um2netcdf.py

```diff
@@ -651,6 +662,379 @@ def test_fix_units_do_nothing_no_um_units(ua_plev_cube):
         assert ua_plev_cube.units == orig  # nothing should happen as there's no cube.units
 
 
+@pytest.fixture
+def lat_river():
+    # iris DimCoord imitating UM V7.3s river grid. Must have length 180.
```

Comment:
Should the comment be tweaked to indicate this is on a half cell offset grid?
(and on line 678)

Reply:
Ah that's the `_u` and `_v` grids that are offset by half a grid cell. This one is on the "river" grid which is higher resolution (i.e. 1x1 degree rather than 1.25x1.875). I've updated the comments on each of the fixtures to clarify these things.